File: Laba2/src/Speakers.java
/**
 * Class <code>Speakers</code> implements speakers of television
 */
public class Speakers {

    private int volume;
    private int maxVolume;

    /**
     * Constructor
     */
    public Speakers() {
        volume = 10;
        maxVolume = 100;
    }

    /**
     * Constructor
     * 
     * @param v Base volume
     * @param password Max volume
     */
    public Speakers(int v, int mV) {
        volume = v;
        maxVolume = mV;
    }

    /** 
     * Return volume
     * @return int
     */
    public int getVolume() {
        return volume;
    }

    /**
     * Method increase speakers volume
     */
    public void increaseVolume() {
        volume++;

        if (volume > maxVolume) {
            volume = 100;
        }
    }

    /**
     * Method decrease speakers volume
     */
    public void decreaseVolume() {
        volume--;

        if (volume < 0) {
            volume = 0;
        }
    }

    /**
     * Method mute speakers
     */
    public void mute() {
        volume = 0;
    }
}

File: Laba9/src/Speakers.py
class Speakers:

    #Constructor
    def __init__(self, volume = 10, maxVolume = 100):
        self.__volume = volume
        self.__maxVolume = maxVolume

    #Return volume
    def getVolume(self):
        return self.__volume
    
    #Method increase speakers volume
    def increaseVolume(self):
        self.__volume += 1

        if self.__volume > self.__maxVolume:
            self.__volume = self.__maxVolume;

    #Method decrease speakers volume
    def decreaseVolume(self):
        self.__volume -= 1

        if self.__volume < 0:
            self.__volume = 0;